app/modules/encounters/models.py

```diff
@@ -140,9 +140,11 @@ def get_location(self):
         return self.get_edm_data_field('locationId')
 
     def get_point(self):
-        lat = self.get_edm_data_field('decimalLatitude')
-        long = self.get_edm_data_field('decimalLongitude')
-        return f'{lat},{long}' if lat and long else None
+        dec_lat = self.get_edm_data_field('decimalLatitude')
```

Comment:
@naknomum was this the final correct way of doing this or WIP?

Reply:
yes i think this is best.  it is self-documenting in the json, rather than their wonky way of having `"X,Y"` and `[X,Y]` etc make **no sense** in terms of which one is latitude and which one is longitude.  :expressionless: